python/ray/projects/projects.py

```diff
@@ -0,0 +1,38 @@
+import json
+import jsonschema
+import os
+import yaml
+
+
+def find_root(dir):
+    """Find root directory of the ray project.
+
+    Args:
+        dir (str): Directory to start the search in.
+
+    Returns:
+        Path of the parent directory containing the .rayproject or
+        None if no such project is found.
+    """
+    prev, dir = None, os.path.abspath(dir)
+    while prev != dir:
+        if os.path.isdir(os.path.join(dir, ".rayproject")):
```

Comment:
Should it be suffix by `.json` or `.yaml`?

Reply:
This is a directory, which contains a project.yaml (and maybe also a cluster.yaml for the autoscaler).